### ISSUE TITLE
[API] URL-Change: tccna.honeywell.com → tccna.resideo.com

### ISSUE THREAD

**Issue by shemege** [luc-ass/homebridge-evohome#177]
They changing domain from https://tccna.honeywell.com to https://tccna.resideo.com. Certificates don't match.
Change this domain everywhere in homebridge-evohome/lib/evohome.js and plugin will run again.

`[Evohome] Error during login:
 Error: Hostname/IP does not match certificate's altnames: Host: tccna.honeywell.com. is not in the cert's altnames: DNS:TCCNA.resideo.com, DNS:TCCEU.resideo.com, DNS:TCCAP.resideo.com, DNS:TCCNAS.resideo.com, DNS:TCCEUS.resideo.com, DNS:TCCAPS.resideo.com`

**Comment by luc-ass** [comment]
Great catch!

**Comment by luc-ass** [comment]
Okay, I've quickly created a fix in `api-url-change`. Would you be so kind and test it? I can't do it at the moment as I am away from home.

From the docs, changed for this branch:
```
# go to node module directory
cd node_modules
# remove homebridge-evohome and contents
rm -r homebridge-evohome
# recreate the folder
mkdir homebridge-evohome
# clone repo to folder
git clone --single-branch --branch api-url-change https://github.com/luc-ass/homebridge-evohome.git ./homebridge-evohome
# cd into folder
cd homebridge-evohome
# install plugin
npm install
```

**Comment by jdegouw** [comment]
wrong version no.? is it 0.10.2 while 0.11 is already there?

**Comment by jdegouw** [comment]
CONFIRMED IT IS WORKING:
![image](https://github.com/user-attachments/assets/ec185257-647f-4a17-801f-861e9cb4d83d)
`[12/12/2024, 12:25:13] [Evohome] Stopping child bridge (will not restart)...
[12/12/2024, 12:25:13] Got SIGTERM, shutting down child bridge process...
[12/12/2024, 12:25:18] [Evohome] Child bridge process ended
[12/12/2024, 13:50:09] [Evohome] Initializing Evohome platform...
[12/12/2024, 13:50:09] [Evohome] Initializing child bridge 0E:4A:35:69:5C:E2
[12/12/2024, 13:50:11] [Evohome] Launched child bridge with PID 1864467
[12/12/2024, 13:50:11] Loaded 0 cached accessories from cachedAccessories.0E866E59757A.
[12/12/2024, 13:50:15] [Evohome] Loaded homebridge-evohome v0.11.1 child bridge successfully
[12/12/2024, 13:50:15] Loaded 0 cached accessories from cachedAccessories.0E4A35695CE2.
[12/12/2024, 13:50:15] [Evohome] Logging into Evohome...
[12/12/2024, 13:50:16] [Evohome] Logged into Evohome!
[12/12/2024, 13:50:16] [Evohome] You have 1 location(s). This instance will be using Index No 0
[12/12/2024, 13:50:16] [Evohome] You have 2 device(s).
[12/12/2024, 13:50:16] [Evohome] 0: Woonkamer (20.5°)
[12/12/2024, 13:50:16] [Evohome] 1: Serre (18.5°)
[12/12/2024, 13:50:16] Initializing platform accessory 'Woonkamer Thermostat'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Serre Thermostat'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Evohome Away Mode'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Evohome Day Off Mode'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Evohome Heating Off Mode'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Evohome Eco Mode'...
[12/12/2024, 13:50:16] Initializing platform accessory 'Evohome Custom Mode'...
[12/12/2024, 13:50:17] Homebridge v1.8.5 (HAP v0.12.3) (Evohome) is running on port 58845.
[12/12/2024, 13:50:17] `

**Comment by PuzzledUser** [comment]
I came here and discovered this thread after doing a manual search and replace for **tccna.honeywell.com** with **tccna.resideo.com** in **evohome.js** after I discovered a certificate mismatch.

```[12/12/2024, 13:44:33] [Evohome] Error during login:
 Error [ERR_TLS_CERT_ALTNAME_INVALID]: Hostname/IP does not match certificate's altnames: Host: tccna.honeywell.com. is not in the cert's altnames: DNS:TCCNA.resideo.com, DNS:TCCEU.resideo.com, DNS:TCCAP.resideo.com, DNS:TCCNAS.resideo.com, DNS:TCCEUS.resideo.com, DNS:TCCAPS.resideo.com
    at Object.checkServerIdentity (tls.js:297:12)
    at TLSSocket.onConnectSecure (_tls_wrap.js:1507:27)
    at TLSSocket.emit (events.js:315:20)
    at TLSSocket._finishInit (_tls_wrap.js:932:8)
    at TLSWrap.ssl.onhandshakedone (_tls_wrap.js:706:12) {
  reason: "Host: tccna.honeywell.com. is not in the cert's altnames: DNS:TCCNA.resideo.com, DNS:TCCEU.resideo.com, DNS:TCCAP.resideo.com, DNS:TCCNAS.resideo.com, DNS:TCCEUS.resideo.com, DNS:TCCAPS.resideo.com",
```
All good for me now

**Comment by jdegouw** [comment]
The change Lucas made is more friendly. You can now change the URL only once and not on 4 or 5 positions.

**Comment by PuzzledUser** [comment]
> The change Lucas made is more friendly. You can now change the URL only once and not on 4 or 5 positions.

Yes, I agree. I had already done it and couldn't find the Lucas source, just the diff. Good we're back up and working. Thanks both!

**Comment by PuzzledUser** [comment]
Yup, better than the 11 inline changes I made! 🤣

**Comment by luc-ass** [comment]
Okay, back home. Thanks for all the feedback! I'll quickly review the changes and publish an official release later today.

**Comment by behdaad** [comment]
Thanks. I appreciate the time you're putting into this.

**Comment by luc-ass** [comment]
Okay, I've changed a few more things around. I'd appreciate it if you would test once again. #179 otherwise seems ready for publishing.
